drupal/modules/lagoon/lagoon_search/lagoon_search.install

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * @file
+ * Update hooks for lagoon search.
+ */
+
+/**
+ * Ensures that the solr configuration is up-to-date.
+ *
+ * The latest version of solr changes the path and core, we need to ensure that
```

Comment:
can you please replace 'latest' with exact version of the solr.

Also, `solr` is `Solr`